nncf/quantization_metric.py

```diff
@@ -0,0 +1,211 @@
+import numpy as np
+import networkx as nx
+from texttable import Texttable
+from collections import deque
+
+from nncf.quantization.layers import SymmetricQuantizer
+from nncf.utils import get_all_modules_by_type
+from nncf.nncf_network import NNCFNetwork
+from nncf.dynamic_graph.transform_graph import is_nncf_module
+
+
+class BaseMetric:
+    def __init__(self):
+        pass
+
+    def collect(self):
+        pass
+
+    def get_metric_table(self):
+        pass
+
+a = ['NNCFConv2d', 'ReLU', 'NNCFLinear']
+
+class NetworkQuantizationShareMetric(BaseMetric):
+    NAME = 'NetworkQuantizationShare'
+
+    WEIGHTS_RATIO = '% weights'
+    ACTIVATIONS_RATIO = '% activations'
+    TOTAL_RATIO = '% total'
+    TOTAL_WEIGHTS_RATIO = '% total w'
+    TOTAL_ACTIVATIONS_RATIO = '% total a'
+
+    PARAMS = 'params'
+    SYMMETRIC = 'symmetric'
+    ASYMMETRIC = 'assymmetric'
+    BITS = 'num_bits'
+    PER_CHANNEL = 'per_channel'
+    SIGNED = 'signed'
+
```

Comment:
Append a `_STR` to all static attribute names that contain a certain string representation for visualization purposes.